src/components/FormInput/index.js

```diff
@@ -6,13 +6,21 @@ const handleChange = action => ev => {
     action(ev.target.value)
 }
 
-const FormInput = ({ isFunny, className, desc, value, onChange, name }) => (
+const FormInput = ({
+    isFunny,
+    className,
+    desc,
+    value,
+    onChange,
+    name,
+    type,
```

Comment:
in order to have default values to react components you can either use 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters
or by setting defaultProps on the function.
eg FormInput.defaultProps = {
    type: 'text'
}